src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const marriedLable = person.sex === 'f' ? 'husband' : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {!!person.age && <p className="Person__age">I am {person.age}</p>}
+
+      {person.isMarried ? (
+        <p className="Person__partner">
```

Comment:
`<p className="Person__partner">` in both cases, only the content is different, so don't repeat it

Reply:
class 'person_partner' repeated  in case ' I am  not married ' ?